Run "yarn tool fix-flaws" on /games (#27999)
mdn/content: files/en-us/games/techniques/audio_for_web_games/index.md

It is worth noting that autoplay with sound is allowed if:

- the User has interacted with the domain.

> - on mobile the user has [added the site to their home screen](/en-US/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable).

Reviewer: ```suggestion
- on mobile the user has [made the application installable](/en-US/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable).
```
